Content.Server/Disposal/Unit/EntitySystems/DisposableSystem.cs

```diff
@@ -66,6 +63,12 @@ public void ExitDisposals(EntityUid uid, DisposalHolderComponent? holder = null,
                     duc.Container.Insert(entity, EntityManager, xform, meta: meta);
                 else
                     xform.AttachToGridOrMap();
+
+                if (EntityManager.TryGetComponent(entity, out IPhysBody? physics))
+                {
+                    physics.CanCollide = true;
```

Comment:
Wakebody should already be setting this to true, also does it take in physicscomponent as a resolve?